.eslintrc.js

```diff
@@ -27,6 +27,7 @@ module.exports = {
     "react/prop-tyes": "off",
     "react/jsx-filename-extension": "off",
     "react/jsx-props-no-spreading": "off",
+    "react/prefer-stateless-function": "off",
```

Comment:
i don't think we should turn this off. We do prefer stateless functions wherever possible.